Readme.md

```diff
@@ -135,7 +135,7 @@ app.use(async (ctx, next) => {
 ```
 
 The `Context` object also provides shortcuts for methods on its `request` and `response`.  In the prior
-examples,  `ctx.type` can be used instead of `ctx.request.type` and `ctx.accepts` can be used
+examples,  `ctx.type` can be used instead of `ctx.response.type` and `ctx.accepts` can be used
```

Comment:
`ctx.type` is delegate to `ctx.request.type` and `ctx.type=` is delegate to `ctx.response.type=`